Terminal.Gui/Core/Toplevel.cs

```diff
@@ -296,17 +297,21 @@ internal void PositionToplevels ()
 		private void PositionToplevel (Toplevel top)
 		{
 			EnsureVisibleBounds (top, top.Frame.X, top.Frame.Y, out int nx, out int ny);
-			if ((nx != top.Frame.X || ny != top.Frame.Y) && top.LayoutStyle != LayoutStyle.Computed) {
-				top.X = nx;
-				top.Y = ny;
+			if ((nx != top.Frame.X || ny != top.Frame.Y) && top.LayoutStyle == LayoutStyle.Computed) {
+				if (top.X is Pos.PosAbsolute && top.Bounds.X != nx) {
+					top.X = nx;
+				}
+				if (top.Y is Pos.PosAbsolute && top.Bounds.Y != ny) {
```

Comment:
This is a good example of what I mean above:

Before long someone's going to see this and say "Hey, if I can ask `Y` if it is absolute, why can't I also ask it if it is `PosFactor`? Then they'll want both to be `public`. Etc...

I have a feeling (but I could be wrong) @migueldeicaza  was very intentional in making these classes `private`. 

The good news on this example is at least `Toplevel` is a very "internal" class very tied to `View` (nobody can build their own version of `Toplevel` that replaces it).

Reply:
@tig I did better, these classes should continue to be `private `rather than `internal`. So I created a PR#778 to remove them.